cafebabel/static/css/styles.css

```diff
@@ -1591,13 +1624,43 @@ textarea#body {
 }
 
 .article-language-selector li span {
-  color: #bde513;
+  background-color: #00283c;
+  color: #fff;
 }
 
 .article-language-selector li.translated-language a {
   color: #00283c;
 }
 
+.article-language-selector li.original-language a {
+  background-color: #bde513;
+  color: #00283c;
+  position: static;
+}
+
+.article-language-selector li.original-language a:hover {
+  background-color: #b4db13;
+}
+
+.article-language-selector li.original-language a:before {
+  content: 'Original language';
```

Comment:
This is inaccessible and will be untranslatable; it should be in the HTML.
As the attribute of the `a` tag maybe? (`<a title="Original language"…>`)